### PR TITLE
Minor Python code health improvements

### PR DESCRIPTION
**Pull request by frankfqchen**
replace is with == for len()

### CODE REVIEW

**Comment by frankfqchen** [comment]
I signed it!

File: tensorflow/python/saved_model/builder.py [hunk 1]
```diff
@@ -143,7 +143,7 @@ def _save_and_write_assets(self, assets_collection_to_add=None):
     asset_source_filepath_list = self._save_assets(assets_collection_to_add)
 
     # Return if there are no assets to write.
-    if len(asset_source_filepath_list) is 0:
+    if len(asset_source_filepath_list) == 0:
```
Comment (rohan100jain) [inline tensorflow/python/saved_model/builder.py]: how about 

if not asset_source_filepath_list ?

**Review by rohan100jain — CHANGES_REQUESTED** [review]

**Review by jart — CHANGES_REQUESTED** [review]
Please `git commit --amend` #4952, #4953, #4954, #4955, #4956, and #4958 into this commit. Then change the description of this commit to "Minor Python code health improvements"

**Comment by jart** [comment]
I'm sad to see this PR closed. These code health improvements would have been nice to have. We hope you'll consider contributing again in the future @frankfqchen.

**Comment by frankfqchen** [comment]
@jart
 i close this cause i don't know how to do "Please git commit --amend #4952, #4953, #4954, #4955, #4956, and #4958 into this commit. Then change the description of this commit to "Minor Python code health improvements"    "

**Comment by frankfqchen** [comment]
@jart   see my new pull request #5025

**Pull request closed without merging**